src/views/products/ProductsIndex.vue

```diff
@@ -63,6 +63,15 @@
         </template>
       </Column>
 
+      <!-- Price -->
+      <Column field="price" header="Unit Price" style="max-width: 14rem">
+        <template #body="{ data }">
+          <InputText :value="getLastPriceValue(data)" disabled
```

Comment:
why disable this input? it will probably be the most convenient way to enter all the prices :)